### DISCUSSION TITLE
Help to get started

### DISCUSSION THREAD

**Discussion by cerchio-pixel** [reactivemarkets/react-financial-charts#475] [Q&A] [answered]
Hello everybody,

Would anybody be so kind to share example code on how to get started? I've been trying to port the storybook code to a CRA app but without success.

Kind Regards

Cerchio

**Comment by cerchio-pixel** [comment]
If anybody needs it... made a simple exampe chart using codesandbox :)

[Codesandbox](https://codesandbox.io/s/react-financial-charts-demo-c88wz)

**Comment by markmcdowell** [comment, marked as answer]
Thanks, sorry for slow response I've been busy!

The storybook examples should be fully contained. e.g.

https://github.com/reactivemarkets/react-financial-charts/blob/master/packages/stories/src/features/StockChart.tsx
